clang/test/CodeGenHLSL/static-local-ctor.hlsl

```diff
@@ -21,7 +21,7 @@ void InitBuf(RWBuffer<int> buf) {
 // CHECK-NEXT: br i1 [[Tmp3]]
 // CHECK-NOT: _Init_thread_header
 // CHECK: init.check:
-// CHECK-NEXT: call void @_ZN4hlsl8RWBufferIiEC1Ev
+// CHECK-NEXT: call void @_ZN4hlsl8RWBufferIiEC1Ejijj
```

Comment:
Same as above - updates the RWBuffer init from the default constructor to the constructor with implicit binding.